packages/react-components/react-dialog/src/components/Dialog/useDialogStyles.ts

```diff
@@ -1,30 +1,41 @@
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { useHasParentContext } from '@fluentui/react-context-selector';
+import { DialogContext } from '../../contexts/dialogContext';
 import type { DialogSlots, DialogState } from './Dialog.types';
 
 export const dialogClassNames: SlotClassNames<DialogSlots> = {
-  root: 'fui-Dialog',
+  overlay: 'fui-Dialog__overlay',
 };
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+  overlay: {
+    position: 'fixed',
+    backgroundColor: 'rgba(0, 0, 0, 0.4)',
+    ...shorthands.inset('0px'),
+  },
+  subDialogOverlay: {
+    backgroundColor: 'transparent',
   },
-
-  // TODO add additional classes for different states and/or slots
 });
 
 /**
  * Apply styling to the Dialog slots based on the state
  */
 export const useDialogStyles_unstable = (state: DialogState): DialogState => {
   const styles = useStyles();
-  state.root.className = mergeClasses(dialogClassNames.root, styles.root, state.root.className);
+  const isSubDialog = useHasParentContext(DialogContext);
```

Comment:
`submenu` is a term but `subdialog` less so IMHO
```suggestion
  const isNestedDialog = useHasParentContext(DialogContext);
```